distribution/docker/ironbank-docker-build-context/build.gradle

```diff
@@ -11,4 +20,8 @@ tasks.register("buildIronBankDockerBuildContext", Tar) {
   // We always treat Iron Bank builds as local, because that is how they
   // are built
   with dockerBuildContext(Architecture.X64, false, DockerBase.IRON_BANK, true)
+
+  into('scripts') {
+    from configurations.transformLog4jJar
```

Comment:
Why are we including this a second time again?

Reply:
We're not - in `distribution/docker/build.gradle`, the jar is copied by the `copy*DockerContext` task. That task also calls the extension method `dockerBuildContext`, like the standalone context subprojects do.

The whole setup here is a bit weird. AFAICT, it exists so that we can generate a Docker context without having to do all the rest of the build. However, and again AFAICT, because we share most of the build context logic via an extension method, it can't (cleanly?) access configurations when called from a subproject. At least, when I tried to just move the jar copy to `dockerBuildContext`, Gradle got angry with me. Suggestions welcome!

Reply:
Yeah, I think really the "solution" here is to ditch the separate projects just for creating the context and just make that a separate task in the associated project. I'm not completely certain why we went with this pattern to begin with other than creating more projects means better parallel execution but copying contexts isn't particularly expensive and `DockerBuild` can already be executed concurrently w/ other tasks in the same project.

Reply:
We may be building things these way partly for the release-manager - we need to coordinate there before changing this.